src/axom/core/docs/sphinx/core_acceleration.rst

```diff
@@ -0,0 +1,51 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core acceleration
+******************************************************
+
+Axom Core provides support to ease the implementation of user applications that
+intend to support execution utilizing accelerators. In order to supply this,
+Axom provides a simple API to access memory and computation, using RAJA
+and Umpire to access accelerators.
+
+.. warning:: 
+   You do need to compile with RAJA and Umpire to fully leverage this Axom functionality. 
+   While both memory management and execution spaces in Axom will operate without these tools, they will offer little.

```

Comment:
Suggestion: The following might be a clearer way to express this point:
```suggestion
   Axom's memory management and execution space APIs have default implementations when Axom is not
   configured with Umpire and RAJA, respectively, and can be used in all Axom configurations.
```

Reply:
changed (thanks)